src/battle_util.c

```diff
@@ -3438,7 +3562,7 @@ u8 AtkCanceller_UnableToUseMove(void)
             if ((gBattleMons[gBattlerAttacker].status1 & STATUS1_PARALYSIS) && (Random() % 4) == 0)
             {
                 gProtectStructs[gBattlerAttacker].prlzImmobility = TRUE;
-                // This is removed in Emerald for some reason
+                // This is removed in Emerald because it is called in BattleScript_MoveUsedIsParalyzed.
```

Comment:
You should revert this change and submit it to Pret's Pokeemerald repository instead.